cnd/src/http_api/action.rs

```diff
@@ -339,4 +341,19 @@ mod test {
             r#"{"type":"ethereum-call-contract","payload":{"contract_address":"0x0a81e8be41b21f651a71aab1a85c6813b8bbccf8","gas_limit":"0x1","network":"ropsten"}}"#
         );
     }
+
+    #[test]
+    fn bitcoin_send_amount_to_address_serializes_correctly_to_json_with_none() {
```

Comment:
:wrench: The name of this test implies that there is some `None` value but there is not.
```suggestion
    fn bitcoin_send_amount_to_address_serializes_correctly_to_json() {
```